tmt-app/src/Components/Breadcrumb.jsx

```diff
@@ -4,9 +4,10 @@ import ForwardArrow from "../Images/Forward.png";
 import { Link } from "react-router-dom";
 
 function Breadcrumb({ previousPage }) {
+  const [category, major, parent, path] = previousPage;
   const breadcrumbs = [
-    { url: `/${previousPage.path}`, label: `${previousPage.parent}` },
-    { label: `${previousPage.major} (${previousPage.category})` },
+    { url: `/${path}`, label: `${parent}` },
+    { label: `${major} (${category})` },
```

Comment:
Okay so the last convo we had I was hoping to get you thinking about this part. You see we're trying to declare a new variable here still rather than passing the breadcrumbs prop directly into the iterator below.


I was thinking the implementation would look more like this:

```jsx

<Breadcrumbs breadcrumbs={[
  {url: "/", label: "Home"},
  {url: "/majors", label: "All Majors"},
  {label: "Education" }
]}>
```

In this implementation, you can drop lines `7-10` we have already created the framework within this component to expect that `breadcrumbs` is going to be an array containing `breadcrumb`s which consists of a url and a label.


```
const [category, major, parent, path] = previousPage;
```

the concept of `category`, `major`, and `parent` don't really have anything to do with "breadcrumbs" but the relationship that the specific instance of this component and its relationships consist of.

if we're aiming for generic components, the only thing this component wants to know is "what is the url" and "what label should it display", "do i have `breadcrumbs` in my given props?"